### PR TITLE
[zh] sync blog: 2023/dlb-connection-balancing/

### PR DESCRIPTION
**Pull request by windsonsea**
Add zh text to this new blog.

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/blog/2023/dlb-connection-balancing/index.md [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+
+显然，它不适用于入口网关，因为入口网关在短时间内接受数千个连接，锁带来的资源成本会导致吞吐量大幅下降。
+
+现在，Envoy 已经集成了 Intel® Dynamic Load Balancing (Intel®DLB) 连接负载均衡，
+以加速在入口网关等高连接数场景中的负载均衡。
+
+## Intel® Dynamic Load Balancing 如何加速 Envoy 中的连接负载均衡 {#how-intel-dlb-accelerate-in-envoy}
+
+Intel DLB 是一个硬件管理的队列和仲裁器系统，连接生产者和消费者。
+它是一个 PCI 设备，预期安装在服务器 CPU 的 [uncore](https://en.wikipedia.org/wiki/Uncore) 中，
```
Comment (wilsonwu) [inline content/zh/blog/2023/dlb-connection-balancing/index.md:38]:
```suggestion
它是一个 PCI 设备，预期安装在服务器 CPU 的 [Uncore](https://en.wikipedia.org/wiki/Uncore) 中，
```

File: content/zh/blog/2023/dlb-connection-balancing/index.md [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+$ kubectl logs -n istio-system ${POD} | grep dlb
+2023-05-05T06:16:36.921299Z     warning envoy config external/envoy/contrib/network/connection_balance/dlb/source/connection_balancer_impl.cc:46        dlb device 0 is not found, use dlb device 3 instead     thread=35
+2023-05-05T06:37:45.974241Z     debug   envoy connection external/envoy/contrib/network/connection_balance/dlb/source/connection_balancer_impl.cc:269   worker_3 dlb send fd 45 thread=47
+2023-05-05T06:37:45.974427Z     debug   envoy connection external/envoy/contrib/network/connection_balance/dlb/source/connection_balancer_impl.cc:286   worker_0 get dlb event 1        thread=46
+2023-05-05T06:37:45.974453Z     debug   envoy connection external/envoy/contrib/network/connection_balance/dlb/source/connection_balancer_impl.cc:303   worker_0 dlb recv 45    thread=46
+2023-05-05T06:37:45.975215Z     debug   envoy connection external/envoy/contrib/network/connection_balance/dlb/source/connection_balancer_impl.cc:283   worker_0 dlb receive none, skip thread=46
+{{< /text >}}
+
+有关 Istio 入口网关的更多细节，请访问
+[Istio 入口网关官方文档](/zh/docs/tasks/traffic-management/ingress/ingress-control/).
```
Comment (wilsonwu) [inline content/zh/blog/2023/dlb-connection-balancing/index.md:325]:
```suggestion
[Istio 入口网关官方文档](/zh/docs/tasks/traffic-management/ingress/ingress-control/)。
```

**Review by wilsonwu — COMMENTED** [review]
LGTM, leave more time for other reviewers, if no more suggestion let's merge it later.

File: content/zh/blog/2023/dlb-connection-balancing/index.md [hunk 3]
```diff
@@ -0,0 +1,325 @@
+---
+title: "在 Istio 中使用加速卸载连接负载均衡"
+description: "在 Istio 网关中使用 DLB 连接负载均衡配置来加速连接均衡。"
+publishdate: 2023-06-08
```
Comment (hanxiaop) [inline content/zh/blog/2023/dlb-connection-balancing/index.md:4]: BTW just saw this https://github.com/istio/istio.io/pull/13707

Comment (windsonsea) [reply content/zh/blog/2023/dlb-connection-balancing/index.md:4]: OK, wait for that merged and then update this PR.

File: content/zh/blog/2023/dlb-connection-balancing/index.md [hunk 4]
```diff
@@ -0,0 +1,325 @@
+---
+title: "在 Istio 中使用加速卸载连接负载均衡"
```
Comment (daixiang0) [inline content/zh/blog/2023/dlb-connection-balancing/index.md:2]:
```suggestion
title: "在 Istio 中使用硬件卸载加速连接负载均衡"
```

File: content/zh/blog/2023/dlb-connection-balancing/index.md [hunk 5]
```diff
@@ -0,0 +1,325 @@
...
+description: "在 Istio 网关中使用 DLB 连接负载均衡配置来加速连接均衡。"
```
Comment (daixiang0) [inline content/zh/blog/2023/dlb-connection-balancing/index.md:3]:
```suggestion
description: "在 Istio 网关中使用 DLB 加速连接负载均衡。"
```

File: content/zh/blog/2023/dlb-connection-balancing/index.md [hunk 6]
```diff
@@ -0,0 +1,325 @@
...
+publishdate: 2023-08-08
+attribution: "Loong Dai (Intel); Translated by Michael Yao (DaoCloud)"
+keywords: [Istio, DLB, gateways]
+---
+
+## 什么是连接负载均衡？ {#what-is-clb}
```
Comment (daixiang0) [inline content/zh/blog/2023/dlb-connection-balancing/index.md:9]: I do not know clb if I do not read the en blog, could we use full spelling?

File: content/zh/blog/2023/dlb-connection-balancing/index.md [hunk 7]
```diff
@@ -0,0 +1,325 @@
...
+
+入口网关被期望尽快地处理尽可能多的数据，因此Intel DLB连接负载均衡使用无序队列。
+
+## 如何在 Istio 中使用 Intel DLB 连接负载均衡 {#how-to-use-intel-dlb-in-istio}
+
+在 1.17 版本发布中，Istio 正式支持 Intel DLB 连接负载均衡。
+
+以下步骤展示了如何在 Istio
+[入口网关](/zh/docs/tasks/traffic-management/ingress/ingress-control/)中使用
+Intel DLB 连接负载均衡，在一个 SPR（Sapphire Rapids）机器上，假设 Kubernetes 集群正在运行。
```
Comment (daixiang0) [inline content/zh/blog/2023/dlb-connection-balancing/index.md:75]:
```suggestion
Intel DLB 连接负载均衡，在一个Kubernetes 集群正常运行的 SPR（Sapphire Rapids）机器上。
```

File: content/zh/blog/2023/dlb-connection-balancing/index.md [hunk 8]
```diff
@@ -0,0 +1,325 @@
...
+      value:
+        connection_balance_config:
+            extend_balance:
+              name: envoy.network.connection_balance.dlb
+              typed_config:
+                "@type": type.googleapis.com/envoy.extensions.network.connection_balance.dlb.v3alpha.Dlb
+EOF
+{{< /text >}}
+
+预计如果您检查入口网关 Pod `istio-ingressgateway-xxxx` 的日志，您将看到类似以下的日志条目：
```
Comment (daixiang0) [inline content/zh/blog/2023/dlb-connection-balancing/index.md:257]:
```suggestion
如果您检查入口网关 Pod `istio-ingressgateway-xxxx` 的日志，您将看到类似以下的日志条目：
```

File: content/zh/blog/2023/dlb-connection-balancing/index.md [hunk 9]
```diff
@@ -0,0 +1,325 @@
...
+
+{{< text bash >}}
+$ echo "$HOST httpbin.example.com" >> /etc/hosts
+$ curl -s -I "http://httpbin.example.com:${PORT}/status/200"
+HTTP/1.1 200 OK
+server: istio-envoy
+...
+{{< /text >}}
+
+访问还未显式暴露的任何其他 URL。您应看到一个 HTTP 404 错误：
```
Comment (daixiang0) [inline content/zh/blog/2023/dlb-connection-balancing/index.md:291]:
```suggestion
访问还未显式暴露的任何其他 URL，您应看到一个 HTTP 404 错误：
```

**Comment by windsonsea** [comment]
Thanks for your careful review 😃

**Review by daixiang0 — APPROVED** [review]
LGTM for contents.

File: content/zh/blog/2023/dlb-connection-balancing/index.md [hunk 10]
```diff
@@ -0,0 +1,325 @@
...
+负载均衡队列有三种类型：
+
+- 无序：适用于多个生产者和消费者。任务的顺序不重要，
+  每个任务都分配给负载最小的处理器核心。
+- 有序：适用于多个生产者和消费者，任务的顺序很重要。
+  当多个任务由多个处理器核心处理时，它们必须按照原始顺序重新排列。
+- 原子：适用于多个生产者和消费者，任务按照一定的规则分组。
+  这些任务使用相同的资源集进行处理，并且组内任务的顺序很重要。
+
+入口网关被期望尽快地处理尽可能多的数据，因此Intel DLB连接负载均衡使用无序队列。
```
Comment (my-git9) [inline content/zh/blog/2023/dlb-connection-balancing/index.md:67]:
```suggestion
入口网关被期望尽快地处理尽可能多的数据，因此 Intel DLB 连接负载均衡使用无序队列。
```

**Review by my-git9 — COMMENTED** [review]
/test lint

**Comment by ericvn** [comment]
Fix for lint test coming.

**Review by hanxiaop — APPROVED** [review]

**Review by my-git9 — APPROVED** [review]
LGTM

**Review by wilsonwu — APPROVED** [review]

**Pull request merged**